src/spiritual/spiritual-edbml/edbml@wunderbyte.com/security/edbml.Security.js

```diff
@@ -56,6 +57,18 @@ edbml.Security = (function using(safeelm, safemap, unsafexp) {
 			"'": '&#39;'
 		};
 	})(),
+	/*
+	 * Replace safe characters in innerHTML
+	 * Only & is a safe character right now
+	 * May be we need more later, depends on
+	 * the requirment of the user
+	 */
+	function safedecode(s) {
```

Comment:
Can you please change `s` to `string` to be consistent?

Reply:
🆗